src/test/java/hw4/FailTest.java

```diff
@@ -0,0 +1,61 @@
+package hw4;
+
+import hw4.pages.MainPage;
+import io.qameta.allure.Feature;
+import io.qameta.allure.Stories;
+import io.qameta.allure.Story;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.testng.ITestContext;
+
+import java.io.IOException;
+
+import static hw4.steps.MainPageSteps.*;
+
+public class FailTest {
+
+    protected WebDriver driver;
+    protected MainPage mainPage;
+    String driverPath = "C:/dev/chromedriver/chromedriver.exe";
```

Comment:
the hardcoded path is not good practice (same as magic constants)
.properties file is a better place 
And to eliminate problems with different driver location `io.github.bonigarcia:webdrivermanager` can be a good solution here

Reply:
Good point! Changed with webdrivermanager.